src/routes/Profile/EducationHistory/EducationHistory.tsx

```diff
@@ -80,39 +92,33 @@ const EducationHistory = () => {
     >
       <Title value={title} />
       <h1 className="mb-5 text-heading-2 sm:text-heading-1">{title}</h1>
-      <p className="mb-8 text-body-lg">{t('profile.education-history.description')}</p>
+      <p className="mb-9 text-body-lg">{t('profile.education-history.description')}</p>
       <ExperienceTable
         mainColumnHeader={t('education-history.education-or-degree')}
+        addNewLabel={t('education-history.add-new-education')}
+        addNewNestedLabel={t('education-history.add-new-degree')}
         rows={rows}
-        onRowClick={handleRowClick}
+        onAddClick={() => setIsWizardOpen(true)}
```

Comment:
After creating a koulutus, "Lisää uusi tutkinto" button produces an error instead of opening anything:
![image](https://github.com/user-attachments/assets/8569a6b0-5320-4c01-abae-e3b7e686e171)